assets/src/dashboard/app/views/shared/telemetryBanner.js

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { TypographyPresets } from '../../../components';
+import { Close as CloseSVG } from '../../../icons';
+import { ICON_METRICS } from '../../../constants';
+import useTelemetryOptIn from './useTelemetryOptIn';
+
+const Banner = styled.div.attrs()`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 161px;
+  margin: 0 20px;
+  padding-top: 24px;
+  background-image: url('/wp-content/plugins/web-stories/assets/images/analytics-banner-bg.png');
```

Comment:
nit, you could import this above and reference it in the url if you want

Reply:
This was wrong anyway, hardcoded URLs don't work. Updated it.